sketch.js

```diff
@@ -0,0 +1,47 @@
+var colony;
+var nest;
+var supply = [];
+
+function setup() {
+  createCanvas(windowWidth, windowHeight);
+  colony = new Colony();
+  nest = new Nest(100,height/6, 200, radians(45));
```

Comment:
There are a lot of "magic numbers" in this setup process. These should be pulled out into appropriately named variables so someone reading the code know what each piece's purpose is